src/pdn/src/straps.cpp

```diff
@@ -134,7 +134,7 @@ bool Straps::checkLayerOffsetSpecification(bool error) const
           utl::PDN,
           185,
           "Insufficient width ({} um) to add straps on layer {} in grid \"{}\" "
-          "with total strap width {} um and offset {} um.",
+          "with total strap width {:.1f} um and offset {:.1f} um.",
```

Comment:
did this help?

Reply:
yess for this kind of error "how the numbers are printed, e.g. 6 not 6.0."

Reply:
@maliberty is there a way to make the printing of floats uniform with the logger or maybe should we review the current messages and add the precision to each and start enforcing the use of precision on future messages?

Reply:
I'm missing the context - what is varying?

Reply:
Depending on the OS/compiler version the output of floats with `{}` is undefined and varies between `6` and `6.0`, thus causing the test to fail when diffing the files.